Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/connected_components.h

```diff
@@ -986,6 +1011,9 @@ void split_connected_components_impl(FIMap fim,
  *   \cgalNPBegin{halfedge_index_map}
  *     a property map containing a unique index for each halfedge initialized 0 to `num_halfedges(pm)`
  *   \cgalNPEnd
+ *  \cgalParamBegin{face_patch_map} a property map with the patch id's associated to the
+     faces of `pm`. Instance of a class model of `ReadPropertyMap`.
```

Comment:
```suggestion
     faces of `pm`. Instance of a class model of `ReadPropertyMap`. If not provided, an internal map will be filled with a call to `connected_components()` with `edge_is_constrained_map()` (if provided).
```

Reply:
You might want to add something similar in the function doc too.

Reply:
If I also add it to the function, it will be said twice, and then somebody else will tell me to remove it.